.github/workflows/publish.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow is triggered three ways:
```

Comment:
Is it safe to use their actions? Is this something we should be removing from a fork?

Reply:
I'd regard it as fairly safe. But as these actions don't actually work on our fork, I've just disabled them for now. We don't use them for building.

![image](https://github.com/seequent/dateutil/assets/26261951/cc36fc49-1b69-4212-b526-50f75fcdc852)